tuftool/src/source.rs

```diff
@@ -56,6 +56,18 @@ pub(crate) fn parse_key_source(input: &str) -> Result<Box<dyn KeySource>> {
         .base_url(Some(&pwd_url))
         .parse(input)
         .context(error::UrlParse { url: input })?;
+    dbg!(&url);
+    dbg!(format!("{}:{}", url.scheme(), url.path()));
+
+    // If we are on on windows , the scheme passed might be the drive letter, so check if the file exists before checking url scheme.
```

Comment:
We can create a new Issue for that (after we finish this one, I'm bad doing much at the same time), I remember writing that because tests failed and tests call `tuftool` with the raw Windows path and that the `url` crate interpreted that as a prefix, and didn't think a lot about it.

IMO, I think that raw path support should be added in both platforms because it's established in other CLI tools, but again, I think that discussing this in another Issue is a good idea.

Reply:
Yeah, it might feel 'wrong' to submit a PR where some tests are failing. But in this case, since we're working on support for a new platform, and since we have multiple Rust crates in the repo, I think it's fine if `tuftool` tests are failing for now.

I've opened #347 and #348 for tracking, and I think we should back out the `tuftool` changes from this PR.